lib/dep-types.js

```diff
@@ -15,20 +15,15 @@ var depTypes = module.exports = function (depName, pkg) {
     from = pkg.optionalDependencies[depName];
   }
 
-  if (pkg.peerDependencies && pkg.peerDependencies[depName]) {
-    type = depTypes.PEER;
-    from = pkg.peerDependencies[depName];
-  }
+  var bundled = !!(pkg.bundleDependencies &&
```

Comment:
any reason this was moved abode the `if (pkg.dependencies ..` ?

Reply:
None I can think of, let me move it back